README.rst

```diff
@@ -40,3 +40,12 @@ See help from python module:
     to the tj3-file. The list of tasks needs to be embedded in a complete tj3-file. See the
     `TaskJuggler website <http://taskjuggler.org/>`_ for more details.
 
+.. note::
+
+    To properly include closed JIRA tickets, your `--query` should order by resolution date.
+
+    Example: `(resolution !=  Invalid OR resolution = Unresolved) ORDER BY resolutiondate ASC, rank ASC`
```

Comment:
Can we rather add this by ourselves, or maybe at least check if it includes this?

Reply:
Removed this outdated note in da2a60de9d12552a5004631aae21f3a9c21659ba.